types/koa-generic-session/index.d.ts

```diff
@@ -1,64 +1,59 @@
-// Type definitions for koa-generic-session 1.x
+// Type definitions for koa-generic-session 2.x
 // Project: https://github.com/koajs/generic-session
-// Definitions by: Nick Simmons <https://github.com/nsimmons>, Ragg <https://github.com/Ragg->
+// Definitions by: Nick Simmons <https://github.com/nsimmons>, Ragg <https://github.com/Ragg->, Javier Garcia <https://github.com/javiertury>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.7
 
 import * as Koa from 'koa';
+import * as Cookies from 'cookies';
```

Comment:
```ts
import * as Cookies from 'cookies';
```
is there such dependency n `koa-generic-session`?
Just pointing on this, as this will results in additional types installed (dependences of @types/cookies). I understand why you want this type

Reply:
The reason why I introduced cookies is because `koa-generic-session` expects the session cookie to be managed by `cookies`.

`options.cookies` as described in the [options documentation](https://github.com/koajs/generic-session#options), must match the structure defined by the package `cookies`. Note that `expressjs/cookies` resolves to `pillarjs/cookies` and contain the source of `cookies`.

![image](https://user-images.githubusercontent.com/1520320/140435486-fd542be5-01e3-41bc-893e-005d38c0c13d.png)

Internally, `koa-generic-session` [implicitly depends](https://github.com/koajs/generic-session/blob/202fa7d9614da8b64fc8482f233868ffb10d063e/src/session.js#L32) on that definition.

Although `koa-generic-session` doesn't list `cookies` as a dependency, it depends on the API of that package.

Reply:
OK, this looks ok for me, you will be the implementation ownerl, thx!